Extensions/Spine/spineruntimeobject-pixi-renderer.ts

```diff
@@ -0,0 +1,127 @@
+namespace gdjs {
+  import PIXI_SPINE = GlobalPIXIModule.PIXI_SPINE;
+
+  export class SpineRuntimeObjectPixiRenderer {
+    _object: gdjs.SpineRuntimeObject;
+    _spine!: PIXI_SPINE.Spine;
+    private _isAnimationCompelete = true;
+
+    get isAnimationCompelete() {
+      return this._isAnimationCompelete;
+    }
+
+    get isUpdatable() {
+      return this._spine.autoUpdate;
+    }
+
+    set isUpdatable(isUpdatable: boolean) {
+      this._spine.autoUpdate = isUpdatable
+    }
+
+    /**
+     * @param runtimeObject The object to render
+     * @param instanceContainer The container in which the object is
+     */
+    constructor(
+      runtimeObject: gdjs.SpineRuntimeObject,
+      private instanceContainer: gdjs.RuntimeInstanceContainer
+    ) {
+      this._object = runtimeObject;
+
+      this.constructSpine();
+      this.updateTimeScale();
+      this.updatePosition();
+      this.updateAngle();
+      this.updateOpacity();
+      this.updateScale();
+
+      instanceContainer
+        .getLayer('')
+        .getRenderer()
+        .addRendererObject(this._spine, runtimeObject.getZOrder());
+    }
+
+    getRendererObject() {
+      return this._spine;
+    }
+
+    onDestroy() {
+      this._spine.destroy();
+    }
+
+    updateTimeScale() {
```

Comment:
Be consistent in return types - either specify in every case or remove everywhere

Reply:
done